.travis.yml

```diff
@@ -38,57 +38,57 @@ dist: trusty
 matrix:
   include:
     # Rails dev / 6 builds >= 2.4.4
-    - rvm: 2.6.3
+    - rvm: 2.6.6
```

Comment:
May I ask why not to use just `rvm: 2.6` that would require less manual updates. It seems [in the documentation](https://docs.travis-ci.com/user/languages/ruby/#specifying-ruby-versions-and-implementations) that it should work.

Reply:
We have found that unreliable in the past, it works for some of the Rubies but can return older Rubies than the current release readily, and wasn't available for all Rubies.